src/main/java/datawave/query/util/AllFieldMetadataHelper.java

```diff
@@ -1588,13 +1618,25 @@ Map<String,Map<String,FieldIndexHole>> findHoles() throws IOException {
             return getImmutableFieldIndexHoles();
         }
         
-        private Date getBaseDate(long ts) {
+        private Date getPreviousDay(long ts) {
```

Comment:
why the shift from current day to previous?

Reply:
This is the result of my comments on the other MR here: https://github.com/NationalSecurityAgency/datawave/pull/2717#issuecomment-2628516416